airflow/www/views.py

```diff
@@ -4236,18 +4236,11 @@ class CustomViewMenuModelView(ViewMenuModelView):
     ]
 
 
-class CustomUserDBModelView(UserDBModelView):
-    """Customize permission names for FAB's builtin UserDBModelView."""
+class CustomUserCommonModelView():
+    """Customize permission names for FAB's builtin UserDBModelView. Common settings for all providers user views"""
```

Comment:
Comment isn't right anymore.

Reply:
thanks @ashb , i will update

Reply:
all users views for providers inherit from the common class that lists the crud permissions, so all of them will now have the add/delete/edit buttons available